docs/source/transformer/enum_cleaner.rst

```diff
@@ -1,7 +1,7 @@
-Enumeration-based Cleaner
```

Comment:
You mixed up the null_cleaner and enum_cleaner

Reply:
Fixed.